src/app/elements/header.tsx

```diff
@@ -6,15 +6,15 @@ export default function Header() {
         className="border-b lg:static border p-4"
         href="./"
       >
-      Boendeparkering i Bergshamra
+        Boendeparkering i Bergshamra
       </a>
       <a
         className="sm:ml-auto lg:pointer-events-auto pointer-events-none"
         href="https://www.johanehrenfors.se"
         target="_blank"
         rel="author"
       >
-      Skapad av Johan Ehrenfors
+        Skapad av Johan Ehrenfors
```

Comment:
Up to you, but maybe "Skapat av Johan & Francis"? Feels wrong to leave your name out from the main page!